src/views/PurchaseOrderReview.vue

```diff
@@ -51,13 +51,25 @@
           </ion-item>
         </div>
       </div>
-      <div v-if="segmentSelected === 'all'">
-        <PurchaseOrderDetail :purchaseOrders="purchaseOrders" :itemsByPoId ="purchaseOrders.parsed"  />
+      <div v-if="searchedProduct?.pseudoId">
+        <PurchaseOrderDetail :purchaseOrders="purchaseOrders" :itemsByPoId ="{[searchedProduct?.orderId]: [searchedProduct]}"  />
       </div>
-      <div v-for="(po, poId) in purchaseOrders.parsed" :key="poId" >
-        <PurchaseOrderDetail v-if="segmentSelected === poId" :itemsByPoId="{[poId]: po}" :purchaseOrders="purchaseOrders" />
+
+      <div class="empty-state" v-else-if="queryString">
```

Comment:
Are there any specific reasons for using the 'empty-state' class instead of Ionic's CSS utility class for center alignment?